README.md

```diff
@@ -157,6 +157,11 @@ docker-compose -f docker-dev.yml up
 
 When finished, use `docker-compose down` to gracefully stop the running containers.
 
+In order to not get Postgres connection error upon signing a submission, you may want to start up the keycloak image as well.
+```shell
+docker-compose up keycloak
+```
+Not having the keycloak image running will produce an error in the console, but will not cause the platform to crash.
```

Comment:
Is there a way to receive the signature email when running the platform from sbt? Like, if I were running a maildev image, could I get the email?

Reply:
Yes, everything should work properly if all the necessary images are running.

Reply:
Could you add instructions for that in the README?

Reply:
Update: Just realized that wouldn't be possible.  We retrieve the email addresses from Keycloak, and the user has to be verified in order for them to receive the submission email.  In a local `sbt` setup, the query for users would return an empty list.  It would theoretically be possible to manually insert users into the Keycloak database for purposes of testing, but that seems like too much trouble in comparison to the full docker-compose setup.

Reply:
Thanks for looking into it! That makes sense.